packages/block-editor/src/components/list-view/branch.js

```diff
@@ -99,6 +99,7 @@ function ListViewBranch( props ) {
 		shouldShowInnerBlocks = true,
 		isSyncedBranch = false,
 		showAppender: showAppenderProp = true,
+		prioritizedInserterBlocks,
```

Comment:
To avoid prop drilling, could we add `prioritizedInserterBlocks` to the list view context around here? https://github.com/WordPress/gutenberg/blob/55e8682d9dd989536699ab819ae9fed9c843c933/packages/block-editor/src/components/list-view/index.js#L197

That way the appender can know about it (and grab it from the list view context), rather than the branches needing to be aware of it.